config/src/main/java/org/springframework/security/config/web/server/ServerHttpSecurity.java

```diff
@@ -3088,7 +3101,7 @@ private ServerLogoutHandler createLogoutHandler() {
 				this.DEFAULT_LOGOUT_HANDLER.setSecurityContextRepository(securityContextRepository);
 			}
 			if (this.logoutHandlers.isEmpty()) {
-				return null;
+				return this.DEFAULT_LOGOUT_HANDLER;
```

Comment:
Sorry I missed this earlier. Will you please place this polish in a separate commit so that the feature commit contains only what is needed?

Also I believe at that point `createLogoutHandler()` cannot return `null`; as such, the null check in `configure` is unnecessary; would you please update that as part of this cleanup?

Reply:
Yes, I think that's fine.

Reply:
@jzheaux I'm not sure what's happening.  I'm trying to revert this, and I'm able to commit the change.  but as soon as I squash the commits, the change goes away.  I'm still working on it.